spec/factories/organizations.rb

```diff
@@ -1,5 +1,11 @@
 FactoryBot.define do
   factory :organization do
     name { Faker::Company.unique.name }
+
+    factory :organization_with_device_group do
+      after(:create) do |organization|
+        create(:device_group, organization: organization)
```

Comment:
does organization have an association for this?

Reply:
we have these associations: 
![image](https://user-images.githubusercontent.com/45994115/58497857-8562f800-8185-11e9-911f-a38ccdb1dc2e.png)

Reply:
hm.. that's interesting.. how come that association existed even the model
kk, looks nice